gatsby-node.js

```diff
@@ -8,12 +8,12 @@ exports.pluginOptionsSchema = ({ Joi }) => {
     cloudName: Joi.string().required(),
     apiKey: Joi.string().required(),
     apiSecret: Joi.string().required(),
-    resourceType: Joi.string().required(),
-    type: Joi.string().required(),
-    maxResults: Joi.integer().required(),
-    tags: Joi.boolean().required(),
-    prefix: Joi.string().required(),
-    context: Joi.boolean().required(),
+    resourceType: Joi.string().default(image),
```

Comment:
The default needs to be string, ie `resourceType: Joi.string().default("image"),` same for type.

Reply:
👍